### DISCUSSION TITLE
Adding Multiple Relation Find Functionality

### DISCUSSION THREAD

**Discussion by KrisPett** [SeaQL/sea-orm#2205] [Q&A]
I'm currently working on a project that involves using the find_also_related method to retrieve related data in my Rust application. However, I encountered an issue where the method doesn't support multiple relations simultaneously.

```rust
let images = Houses::find_by_id(&id)
    .find_also_related(entity::brokers::Entity)
    .find_also_related(entity::house_images::Entity)
    .all(db_conn)
    .await
    .map_err(|err| Error::from(CustomErrors::DatabaseError(err)))?;
```

As you can see, I'm attempting to find related data for both brokers and house images, but it seems that the find_also_related method only supports one relation at a time.

1. Are there any plans to add support for multiple relation finds in the future? If so, is there an estimated timeline for when this feature might be available?

2. In the absence of support for multiple relation finds, are there any alternative solutions or workarounds that you would recommend to achieve a similar result?

I appreciate any insights or suggestions you can provide regarding this matter.

Thank you!

**Comment by tyt2y3** [comment]
Yes, there is a plan. But sorry, no ETA. You can write a generic function that do a custom select with multiple joins.

**Reply by tyt2y3** [reply]
Related: https://github.com/SeaQL/sea-orm/discussions/1502
